modules/1_bastion/templates/create_disk_link.sh

```diff
@@ -1,14 +1,23 @@
-#Scan devices
 sudo rescan-scsi-bus.sh -a -m -r
```

Comment:
Not sure if we need it for vscsi? @bpradipt 
No harm running it but will have some time saved.

Reply:
I don't know if it will be required or not. Let's keep it for now and we can circle back after some experimentation